### PR TITLE
Add support for channel 2 of AdqDigitizer's

### PR DESCRIPTION
**Pull request by JamesWrigley**
I also sneaked in 6d911682 to fix the tests :hand_over_mouth:

### CODE REVIEW

File: extra_foam/pipeline/processors/digitizer.py
Comment (bermudei) [inline extra_foam/pipeline/processors/digitizer.py:1]: maybe just also keeping 'digitizers.channel_1_A.apd.pulseIntegral': 'A'. Just in case they need it

**Comment by takluyver** [comment]
LGTM, thanks.

I haven't totally understood the hget related changes, but they look safe enough; I presume something changed with a new version of the redis client library.

**Review by bermudei — APPROVED** [review]
LGTM. The warning changes look also safe enough as thomas mentioned. Thanks James!

File: extra_foam/pipeline/processors/digitizer.py
Comment (JamesWrigley) [reply extra_foam/pipeline/processors/digitizer.py:1]: Ah that's still in there :) Github just clips the file, if you press the  image button it should become visible.

**Comment by JamesWrigley** [comment]
> I haven't totally understood the hget related changes, but they look safe enough; I presume something changed with a new version of the redis client library.

Yeah, so we're passing enums to the function and those used to be automagically converted to integers, but now it doesn't do that anymore so we have to call `.value` on the enum directly.

**Pull request merged**